Libraries/pbxspec/Sources/Manager.cpp

```diff
@@ -460,10 +511,11 @@ Create(void)
     return std::make_shared <Manager> ();
 }
 
-std::string Manager::
+std::string const &Manager::
 AnyDomain()
 {
-    return "<<domain>>";
+    static std::string any = "<<domain>>";
```

Comment:
Depends on what you mean by "bad". This object is going to be constructed the first time you call the function, which seems fine to me?